apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/tally-template-a/tally-template-a.component.ts

```
@@ -0,0 +1,118 @@
+    ) {
+        this.companySetting$ = this.store.pipe(select(state => state.settings.profile), takeUntil(this.destroyed$));
+    }
+
+    /**
+     * Initializes the component
+     *
+     * @memberof TallyTemplateAComponent
+     */
+    public ngOnInit(): void {
+        this.companySetting$.subscribe(address => {
+            if (address && address.address) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)